python/ray/tests/test_state_api.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+
+import ray
+from ray.experimental.state.api import (
+    list_actors,
+    list_placement_groups,
+    list_nodes,
+    list_jobs,
+    list_workers,
+)
+from ray._private.test_utils import wait_for_condition
+from ray.job_submission import JobSubmissionClient
+
+
+def test_list_actors(shutdown_only):
+    ray.init()
+
+    @ray.remote
+    class A:
+        pass
+
+    a = A.remote()  # noqa
+
+    def f():
+        state = list(list_actors().values())[0]["state"]
+        return state == "ALIVE"
+
+    wait_for_condition(lambda: f())
+    print(list_actors())
```

Comment:
Could we convert `print` to checks since no one is going to manually verify this?

Reply:
Actually, actual verification is already in `wait_for_condition `. I think printing it at the end is good for local development. Since it won't be printed to the CI unless tests failed, I think it'd be okay to have it here.

Reply:
yea, I'm fine as long as we add extra checks in f()